### ISSUE TITLE
Creating your own pickit item

### ISSUE THREAD

**Issue by ixrogue** [kolton/d2bot-with-kolbot#1307]
First and foremost, I’ve searched far and wide for How To’s for making your own custom item for it to pickit. 

First problem, pickit will only search for rare circlets but I need rare diadems,tiaras, and coronets.

Secondly I WISH someone can help us all with what each symbol would mean example what    l l     Means or when do we put the ## or && signs. 

I’m specifically I’m trying to make the helmsets with fcr 2 to class skills with Str res life etc etc.
And also looking for a Foh stick that my bot will ONLY pick up with 3 to combat skills/concentration or 2 to paladin skills with concen/battle hammers

But there’s no guide on how to make these items specifically.

If someone can make a semi depth guide or show me a legit guide to customizing the items you want.

Long story short: please explain each code (how it works/what it does) and how to put them together to make it find exactly what you are looking for

**Comment by ixrogue** [comment]
Helmets*

**Comment by 5noop** [comment]
> But there’s no guide on how to make these items specifically.

Check this https://github.com/blizzhackers/pickits/blob/master/NipGuide.md/#nip-guide

**Comment by DarkHorseDre** [comment]
https://github.com/kolton/d2bot-with-kolbot/issues/1296

**Comment by ixrogue** [comment]
I appreciate it, I guess Kolton already had a foh stick made. It’s weird because it’ll pick up a scepter if it has ONE of the skills of the foh stick.    IE.) 3 to concen but no BH or 3 to conviction and no FOH

Last question I’m assuming I must have those combined skills be (concen + battle hammer ) >= 6 

^ wasnt actual coding just example.

That’s my issue atm it’s only picking up 1 of those types and saving it without combined skills being on scepter

**Comment by DarkHorseDre** [comment]
well if you write [stat1]+[stat2] >= 3, then either stat can be 0, 1, 2, 3
if you write [stat1] == 3 && [stat2] ==3 then both must be 3

read the link Snoop pasted above - it explains what you need

**Comment by Blitz28** [comment]
if you are trying to do a variety of the same type use [type] instead of [name]
[name] == circlet will only pickup circlets but not diadems, tiaras, ect
[type] == circlet will pick up all circlet class items

other examples would be 

[name] == thresher
vs
[type] == polearm

or 

[name] == shako
vs
[type] == helmet
